src/renderer/constants/apiUrls.ts

```diff
@@ -0,0 +1,41 @@
+import { Epoch, Slot, Shard, BLSPubkey, BeaconBlock, uint8, IndexedAttestation } from '@chainsafe/eth2.0-types';
+
+export const API_URL: String = `https://${process.env.HOSTNAME || 'localhost'}:${process.env.PORT || '3000'}`;
+
+/**
+ * Minimal Beacon Node API for Validator
+ *
+ * A minimal API specification for the beacon node, which enables
+ * a validator to connect and perform its obligations on the Ethereum
+ * 2.0 phase 0 beacon chain.
+ */
+
+export const FETCH_NODE_VERSION: string = `${API_URL}/node/version`;
+export const FETCH_GENESIS_TIME: string = `${API_URL}/node/genesis_time`;
+export const POLL_NODE_SYNCING: string = `${API_URL}/node/syncing`;
+export const FETCH_FORK_INFORMATION: string = `${API_URL}/node/fork`;
+
+export const FETCH_VALIDATOR_DUTIES: Function = (validator_pubkeys: BLSPubkey[], epoch: Epoch) => {
+    return `${API_URL}/validator/duties?validator_pubkeys=${validator_pubkeys}&epoch=${epoch}`;
```

Comment:
I'm pretty sure this doesn't work as pubkeys are buffers and should be serialized as array of hex strings.
Something like this might work:
```javascript
const hexPubKeys = validatorPublicKeys.map(key => key.toString("hex"));
const url = `/duties?validator_pubkeys=${JSON.stringify(hexPubKeys)}&epoch=${epoch.toString()}`;
```